src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.scss

```diff
@@ -120,3 +129,15 @@ md-icon {
   color: $delicate;
   font-size: $subhead-font-size-base;
 }
+
+.kd-sparkline {
+  width: 100px;
+  height: 20px;
```

Comment:
Changed in becde9ed735aa9274c1578b0ec70f8a2f9acc812, but with screwy coefficients. Let me know if I can round these out :)